From johnpol@2ka.mipt.ru Sun Jun 18 22:59:02 2006
Date: Mon, 19 Jun 2006 09:59:02 +0400
From: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
To: GregKH <greg@kroah.com>
Subject: W1: remove w1 mail list from lm_sensors.
Message-ID: <20060619055902.GA11750@2ka.mipt.ru>
Content-Disposition: inline

lm_sensors mail list is going to be splitted into separate subdev lists,
so remove w1 from there.

http://lists.lm-sensors.org/pipermail/lm-sensors/2006-June/016507.html

Signed-off-by: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/MAINTAINERS
+++ gregkh-2.6/MAINTAINERS
@@ -3162,7 +3162,6 @@ S:	Maintained
 W1 DALLAS'S 1-WIRE BUS
 P:	Evgeniy Polyakov
 M:	johnpol@2ka.mipt.ru
-L:	lm-sensors@lm-sensors.org
 S:	Maintained
 
 W83L51xD SD/MMC CARD INTERFACE DRIVER
